roboteam_ai/src/skills/KickTo.cpp

```diff
@@ -0,0 +1,36 @@
+#include <roboteam_ai/src/control/PositionUtils.h>
+#include "KickTo.h"
+#include <roboteam_ai/src/world/Field.h>
+#include <roboteam_ai/src/control/numTrees/NumTreePosControl.h>
+#include <roboteam_ai/src/control/BasicPosControl.h>
+#include <roboteam_ai/src/control/ControlUtils.h>
+
+namespace rtt {
+namespace ai {
+
+KickTo::KickTo(string name, bt::Blackboard::Ptr blackboard)
+        :Skill(std::move(name), std::move(blackboard)) {
+}
+void KickTo::onInitialize() {
+    shootPos=properties->getVector2("shootPos");
```

Comment:
never initialize an absolute position in the field.

this changes for fieldsize